src/pages/FridgeDetailPage.js

```diff
@@ -44,56 +36,20 @@ import {
 } from '../globalStyles';
 import { vaccineStrings } from '../localization/index';
 
-const CardText = ({ children, color, small }) => (
-  <Text style={{ color, fontSize: small ? 12 : 50, fontFamily: APP_FONT_FAMILY }}>{children}</Text>
+const NoBreachMessage = () => (
+  <>
+    <NoBreachMan />
+    <Text style={localStyles.noBreachText}>{vaccineStrings.no_breaches}</Text>
+  </>
 );
 
-CardText.defaultProps = { small: false };
-
-CardText.propTypes = {
-  children: PropTypes.node.isRequired,
-  color: PropTypes.string.isRequired,
-  small: PropTypes.bool,
-};
-
-const BreachCard = ({ content, type }) => {
-  const color = type === 'HOT_CUMULATIVE' ? DANGER_RED : COLD_BREACH_BLUE;
-  const Icon = type === 'HOT_CUMULATIVE' ? HotBreachIcon : ColdBreachIcon;
-
-  const header = type.includes('CUMULATIVE')
-    ? vaccineStrings.cumulative_breach
-    : vaccineStrings.consecutive_breach;
-
-  return (
-    <CardDetail
-      headerText={header}
-      Content={content ? <CardText color={color}>{content}</CardText> : <NoBreachMan />}
-      Footer={
-        content ? (
-          <Icon color={color} />
-        ) : (
-          <CardText color={color} small>
-            {vaccineStrings.no_breaches}
-          </CardText>
-        )
-      }
-    />
-  );
-};
-
-BreachCard.defaultProps = {
-  content: null,
-};
-
-BreachCard.propTypes = {
-  content: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
-  type: PropTypes.oneOf([
-    'COLD_CUMULATIVE',
-    'COLD_CONSECUTIVE',
-    'HOT_CONSECUTIVE',
-    'HOT_CUMULATIVE',
-  ]).isRequired,
-};
+const BreachCard = props => (
```

Comment:
The logic was pretty fragile and specific, as evidenced by it being bork IMO. So I nuked it, sorry. I don't think it should be difficult to read some JSX that has been composed on the spot, rather than a highly specific component that generates each permutation. If you look at the render function I think it's easier to know what you're actually getting without having to chunk so much in your head, and you can see what stylesheets are tweaking what (which is pretty much the base approach to styles prescribed in the RN docs).

An underlying theme with components I've removed is difficulty getting consistent behaviour when of applying StyleSheets to get desired layouts.

You've said that you reckon you over extract (abstract out) things; where do you reckon the line is? For me it's not much above `Paper`, but it's definitely hard to quantify.

I agree I should have scrapped it all together, I just felt like keeping the styles out a little, little homage to the original (probably bad reasoning haha).